diff --git a/nvbuild.sh b/nvbuild.sh
index fb3122b..840b04f 100755
--- a/nvbuild.sh
+++ b/nvbuild.sh
@@ -51,7 +51,7 @@ if [[ -z "${TARGETARCH}" ]]; then
   fi
 fi
 
-export TF_CUDA_COMPUTE_CAPABILITIES="local"
+export TF_CUDA_COMPUTE_CAPABILITIES="custom"
 
 while [[ $# -gt 0 ]]; do
   case $1 in
@@ -82,8 +82,9 @@ done
 
 THIS_DIR="$(cd "$(dirname "${BASH_SOURCE[0]}")" >/dev/null && pwd)"
 cd "${THIS_DIR}/tensorflow-source"
-
-if [[ "$TF_CUDA_COMPUTE_CAPABILITIES" == "all" ]]; then
+if [[ "$TF_CUDA_COMPUTE_CAPABILITIES" == "custom" ]]; then
+  TF_CUDA_COMPUTE_CAPABILITIES="7.0,8.0"
+elif [[ "$TF_CUDA_COMPUTE_CAPABILITIES" == "all" ]]; then
   TF_CUDA_COMPUTE_CAPABILITIES="$(${THIS_DIR}/nvarch.sh ${TARGETARCH})"
   if [[ $? -ne 0 ]]; then exit 1; fi
 elif [ "$TF_CUDA_COMPUTE_CAPABILITIES" == "local" ]; then
